vtex-apps/app-store
# Set a target account by query string param

#### What is the purpose of this pull request?

Provide a way to pre-set this input:
![image](https://user-images.githubusercontent.com/3827456/90051358-f371dd00-dcad-11ea-9dbd-57e56cedef83.png)
Using a query string parameter

#### What problem is this solving?

This is useful to handle redirects from the users' admins. That way the store can minimize friction by presetting the final redirection parameter.

#### How should this be manually tested?

Visit [this linked workspace](https://artur--extensions.myvtex.com/google-shopping/r?account=basedevmkp). Change the `account` query string parameter and reload. The text input should change its content.

#### Screenshots or example usage
None

#### Types of changes
